### PR TITLE
Added "Clear all" button to LOI selection part of onboarding flow

### PR DESCRIPTION
**Pull request by nwkotto**
The "Clear all" button clears out existing LOIs on the survey one by one, in quick succession.

### CODE REVIEW

**Comment by nwkotto** [comment]
https://screenshot.googleplex.com/BzdKzuisfAWvfnr

File: web/src/app/pages/main-page-container/main-page/map/map.component.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ export class MapComponent implements AfterViewInit, OnDestroy {
 
   @ViewChild(GoogleMap) map!: GoogleMap;
 
-  @Input() shouldEnableDrawingTools = false;
+  @Input() shouldEnableDrawingTools = true;
```
Comment (DaoyuT) [inline web/src/app/pages/main-page-container/main-page/map/map.component.ts:103]: Did you mean to include this?

File: web/src/app/pages/main-page-container/main-page/main-page.component.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ import {TitleDialogComponent} from './title-dialog/title-dialog.component';
 export class MainPageComponent implements OnInit {
   activeSurvey$: Observable<Survey>;
   subscription: Subscription = new Subscription();
-  shouldEnableDrawingTools = false;
+  shouldEnableDrawingTools = true;
```
Comment (DaoyuT) [inline web/src/app/pages/main-page-container/main-page/main-page.component.ts:44]: Did you mean to include this?

**Review by DaoyuT — CHANGES_REQUESTED** [review]
Looks good in general

File: web/src/app/pages/main-page-container/main-page/map/map.component.ts (hunk 1 above)
Comment (nwkotto) [reply web/src/app/pages/main-page-container/main-page/map/map.component.ts]: Whoops, I did not. Removed.

File: web/src/app/pages/main-page-container/main-page/main-page.component.ts (hunk 2 above)
Comment (nwkotto) [reply web/src/app/pages/main-page-container/main-page/main-page.component.ts]: Whoops, I did not. Removed.

**Review by DaoyuT — APPROVED** [review]
LGTM

**Pull request merged**